tox.ini

```diff
@@ -38,17 +38,12 @@ commands =
 
 [testenv:lint]
 changedir = {toxinidir}
-lint_dirs = tuf/api tuf/ngclient examples tests
+lint_dirs = tuf examples tests
```

Comment:
Did you check if all of the tools recursively check the next folders in `tuf/` - `tuf/ngclient` and `tuf/api`?